test/helpers/mock-github.js

```diff
@@ -5,15 +5,15 @@ const nock = require('nock');
  *
  * @param {Object} [env={}] Environment variables.
  * @param {String} [githubToken=env.GH_TOKEN || env.GITHUB_TOKEN || 'GH_TOKEN'] The github token to return in the authentication response.
- * @param {String} [githubUrl=env.GH_URL || env.GITHUB_URL || 'https://api.github.com'] The url on which to intercept http requests.
+ * @param {String} [githubUrl=env.GITHUB_API_URL || env.GH_URL || env.GITHUB_URL || 'https://api.github.com'] The url on which to intercept http requests.
  * @param {String} [githubApiPathPrefix=env.GH_PREFIX || env.GITHUB_PREFIX || ''] The GitHub Enterprise API prefix.
  * @return {Object} A `nock` object ready to respond to a github authentication request.
  */
 function authenticate(
   env = {},
   {
     githubToken = env.GH_TOKEN || env.GITHUB_TOKEN || 'GH_TOKEN',
-    githubUrl = env.GH_URL || env.GITHUB_URL || 'https://api.github.com',
+    githubUrl = env.GITHUB_API_URL || env.GH_URL || env.GITHUB_URL || 'https://api.github.com',
```

Comment:
it feels a bit odd that we replicate the implementation of env variable parsing here. We basically test our test helper now, instead of the actual implementation. But I don't have a better idea